dask/dataframe/tseries/tests/test_resample.py

```diff
@@ -128,12 +128,29 @@ def test_resample_pads_last_division_to_avoid_off_by_one():
     assert_eq(actual, expected)
 
 
-def test_series_resample_not_implemented():
+def test_resample_does_not_evenly_divide_day():
+    import numpy as np
+
+    p = 100
+    index = pd.date_range(start="2012-01-02", periods=p, freq="H")
```

Comment:
In other places in this test file we create indexes with holes in them:

https://github.com/dask/dask/blob/38d85be356ba7bfc0bf561881525b75b7a5a7e7e/dask/dataframe/tseries/tests/test_resample.py#L29-L30

Can you do something like that in these?

Reply:
Added. Thanks for the review @jsignell